wsmaster/che-core-sql-schema/src/main/resources/che-schema/5.6.0/1__add_exec_agent_where_terminal_agent_is_present.sql

```diff
@@ -1,5 +1,5 @@
 --
--- Copyright (c) 2012-2017 Codenvy, S.A.
+-- Copyright (c) 2012-2019 Codenvy, S.A.
```

Comment:
Just a making sure. Codenvy is acquired by Red Hat, right? Why the latest year is updated?

```suggestion
-- Copyright (c) 2012-2019 Red Hat, Inc.
```
or

```suggestion
-- Copyright (c) 2012-2017 Codenvy, S.A.
-- Copyright (c) 2017-2018 Red Hat, Inc.
```

is better ?